docs/Getting Started/Integration Basics - iOS/index.md

```diff
@@ -0,0 +1,559 @@
+# Integration Basics
+### How SDL Works
+SmartDeviceLink works by sending remote procedure calls (RPCs) back and forth between a smartphone application and the SDL Core. These RPCs allow you to build the user interface, detect button presses, play audio, and get vehicle data, among other things. You will use the SDL library to build your app on the SDL Core.
```

Comment:
```suggestion
```

I don't think this is necessary and contains poor grammar.

Reply:
@renonick87 @joeljfischer I agree that this might not be necessary for the integration guides. If we remove it from the iOS guides I think we should also remove it from the Java and JS docs as well. What do you think

Reply:
I agree

Reply:
I agree as well.